cmd/clabverter/cli/entrypoint.go

```diff
@@ -58,6 +59,12 @@ If not set, clabverter will look for a file named '*.clab.y*ml'`,
 				Required: false,
 				Value:    false,
 			},
+			&cli.StringFlag{
```

Comment:
maybe it'd be easier to just have a bool flag that defaults to whatever our default is?

since we default to prefixed, could be `disablePrefixedNaming` or something better, then dont need the string slice check or any of that extra stuff

Reply:
though maybe I also said the other thing was a string in the config for .... *some reason* so maybe this is smarter for future proofiness

Reply:
yeah, I remember you said you wanna have some FLEXIBILITY and thus a string param was erected

Reply:
*robustness* is key :D, ship it! :D